tests/test_tracing.py

```diff
@@ -110,7 +110,7 @@ def test_trace_attributes_missing(db, monkeypatch):
     )
 
 
-def test_initialise_trace(db):
+def test_initialise_trace1(db):
```

Comment:
```suggestion
def test_initialise_trace(db):
```
looks like a wip that accidentally made it thru